# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../etcnet/etcnet-0.8.5-Makefile-ln.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- etcnet-0.8.5/contrib/Makefile.orig	2007-03-01 21:00:33.000000000 +0200
+++ etcnet-0.8.5/contrib/Makefile	2007-03-01 21:02:02.000000000 +0200
@@ -45,6 +45,6 @@
 
 # 3. Create /sbin links.
 	@install -d $(sbindir)
-	@ln -s ../etc/net/scripts/ifup $(sbindir)/ifup
-	@ln -s ../etc/net/scripts/ifdown $(sbindir)/ifdown
+	@ln -sf ../etc/net/scripts/ifup $(sbindir)/ifup
+	@ln -sf ../etc/net/scripts/ifdown $(sbindir)/ifdown
 	@echo Done!
